soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -2049,54 +2049,54 @@ GetItemID Randomizer::GetItemFromGet(RandomizerGet randoGet, GetItemID ogItemId)
             return ogItemId;
 
         case RG_KOKIRI_SWORD:
-            return GI_SWORD_KOKIRI;
+            return !CHECK_OWNED_EQUIP(EQUIP_SWORD, 0) ? GI_SWORD_KOKIRI : GI_RUPEE_BLUE;
```

Comment:
is there a specific reason you're doing this instead of
```cpp
CHECK_OWNED_EQUIP(EQUIP_SWORD, 0) ? GI_RUPEE_BLUE : GI_SWORD_KOKIRI
```

Reply:
It felt more readable to me, no other real reason. I can invert them all if you'd like.

Reply:
coolcool, it's fine to keep them as is, the `! ?` combo just threw me for a sec